source/renderer/app/components/wallet/backup-recovery/WalletRecoveryPhraseEntryDialog.js

```diff
@@ -130,15 +130,19 @@ export default class WalletRecoveryPhraseEntryDialog extends Component<Props> {
 
```

Comment:
@nikolaglumac @DominikGuzei 
MnemonicWord's `onClick` handler was previously an inline arrow function, which I've replaced with a constant called `handleClick`. This should help with unnecessary and frequent re-rendering of each MnemonicWord component. Previously, every MnemonicWord component would re-render because ReactDOM's diffing algorithm detects the inline arrow function as a new piece of data every time `recoveryPhraseShuffled.map` is executed (which happens each time a mnemonic word is clicked).

This is one of those things people like to argue about, whether it's necessary or not. I've found that if you are mapping over an array and rendering many instances of the same component for each index of the array, it's one of the best use cases. Not trying to be petty, but I remember seeing a task card about preventing unnecessary re-rendering.